beacon-chain/light-client/store.go

```diff
@@ -16,17 +17,20 @@ import (
 	ethpb "github.com/prysmaticlabs/prysm/v3/proto/prysm/v1alpha1"
 )
 
-const ()
-
 type Store struct {
-	BeaconChainConfig             *params.BeaconChainConfig  `json:"beacon_chain_config,omitempty"`
-	FinalizedHeader               *ethpbv1.BeaconBlockHeader `json:"finalized_header,omitempty"`
-	CurrentSyncCommittee          *ethpbv2.SyncCommittee     `json:"current_sync_committeeu,omitempty"`
-	NextSyncCommittee             *ethpbv2.SyncCommittee     `json:"next_sync_committee,omitempty"`
-	BestValidUpdate               *Update                    `json:"best_valid_update,omitempty"`
-	OptimisticHeader              *ethpbv1.BeaconBlockHeader `json:"optimistic_header,omitempty"`
-	PreviousMaxActiveParticipants uint64                     `json:"previous_max_active_participants,omitempty"`
-	CurrentMaxActiveParticipants  uint64                     `json:"current_max_active_participants,omitempty"`
+	BeaconChainConfig *params.BeaconChainConfig `json:"beacon_chain_config,omitempty"`
```

Comment:
Not sure why `lint` didn't capture this.  
I thought all the public field should have a comment 

For `BeaconChainConfig` / `PreviousMaxActiveParticipants` / `CurrentMaxActiveParticipants` / `NextSyncCommittee`, there aren't.

Also, my old experience for Go lint is the comment should starts with the field name?

Reply:
I think it's due to this:

https://github.com/prysmaticlabs/prysm/blob/develop/.golangci.yml#L5

I'll still fix it

Reply:
There's a preset called `comment` that I don't see enabled, maybe that's why?

https://github.com/polymerdao/prysm/blob/develop/.golangci.yml#L13

I addressed your comment here https://github.com/polymerdao/prysm/commit/0ad73eb771146594b69f21eb4fc0339a19b32344